ramls/dataExport.raml

```diff
@@ -58,6 +59,34 @@ resourceTypes:
           body:
             text/plain:
               example: "Internal server error"
+  /quick-export:
+    displayName: Export manager service
+    description: Method to start quick export for MARC records from inventory
+    post:
+      description: Starts the export process
```

Comment:
description: Starts the quick export process

Reply:
updated